package.json

```diff
@@ -5,8 +5,8 @@
     "dev": "yarn start",
     "start": "vuepress dev docs",
     "build": "vuepress build docs",
-    "lint": "prettier docs/guide/*.md",
-    "lint:fix": "prettier docs/guide/*.md --write",
+    "lint": "prettier docs/guide/*.md docs/snippets/*",
+    "lint:fix": "yarn lint --write",
```

Comment:
Huh, what's going on here? Not using `prettier` for `lint:fix` anymore?

Reply:
No I am, I'm just (I suppose) currying it.

`yarn lint -- write` -> `prettier docs/guide*.md docs/snippets/*`

Reply:
Oh, whoops! I misread this.